src/dataunion/DataUnion.ts

```diff
@@ -565,6 +595,30 @@ export class DataUnion {
     }
 
     // Internal functions
+    /** @internal */
+    static async _getBinanceDepositAddress(userAddress: string, client: StreamrClient) {
+        const contracts = new Contracts(client)
+        const adapter = await contracts.getBinanceAdapter()
+        const recipientAddress = (await adapter.binanceRecipient())[0]
+        if (recipientAddress === '0x') { return undefined } // TODO: test this
```

Comment:
I changed this because `== 0` sounds like it at least warrants a comment as to what is expected to "equal to" zero. Please use the exact "zero address answer" if it's known.